perfkitbenchmarker/scripts/database_scripts/file_to_plot.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Plots per second output from STDERR files.
+How to use:
+  Initialize a plotter instance.
+  Add relevant STDERR files using add_file() method.
+  When finished adding files, utilize plot() method to generate a chart.
+  If using launch_mysql_service enabling 'per_second_graphs' feature flag will
+  automatically add files as runs complete and plot after last thread count call
+  finishes.
+"""
+import datetime
+import plot_scatter_points
+import subprocess
+
+DATETIME_FORMAT = '{:%m_%d_%Y_%H_%M_}'
+DATA_INDICATOR_LINE = 'Threads started!'
```

Comment:
put a note here that our parsing depends on sysbench output. and specify the version of sysbench we are using when this code was written.